README.md

```diff
@@ -65,9 +81,17 @@ Executing the systems of this service require different steps:
 > Press the green button next to the `main` method in `Session.java` and the server will be executed:
 ![executionImage](readmeresources/backendexecution.png)
 
+
+
+## How to report a bug
+No sure what to put on this section for now
+
+##  Known bugs
+- N/A
```

Comment:
Put a hyperlink to here https://github.com/ngutn24/RollBuddy/issues?q=is%3Aopen+is%3Aissue+label%3Abug